Security/Exception/LtiToolMessageExceptionHandler.php

```diff
@@ -45,7 +45,9 @@ public function __construct(?ParserInterface $parser = null)
      */
     public function handle(Throwable $exception, Request $request): Response
     {
-        $payload = new LtiMessagePayload($this->parser->parse($request->get('id_token')));
```

Comment:
This was simple enough, not to be split into variables, and keeping it inlined IMO.